src/index.js

```diff
@@ -71,158 +71,188 @@ class App extends React.Component {
+    }));
+  };
 
   // https://stackoverflow.com/questions/21733847/react-jsx-selecting-selected-on-selected-select-option
-  handleSttTypeChange = (event) => {
+  handleSttTypeChange = event => {
     console.log(event.target.name, event.target.value);
     this.setState({ [event.target.name]: event.target.value });
-  }
+  };
 
-  handleExportFormatChange = (event) => {
+  handleExportFormatChange = event => {
     console.log(event.target.name, event.target.value);
     this.setState({ [event.target.name]: event.target.value });
-  }
+  };
 
   exportTranscript = () => {
     // eslint-disable-next-line react/no-string-refs
-    const { data, ext } = this.transcriptEditorRef.current.getEditorContent(this.state.exportFormat);
+    const { data, ext } = this.transcriptEditorRef.current.getEditorContent(
+      this.state.exportFormat
+    );
     this.download(data, `${ this.state.mediaUrl }.${ ext }`);
-  }
+  };
 
   // https://stackoverflow.com/questions/2897619/using-html5-javascript-to-generate-and-save-a-file
-   download = (content, filename, contentType) => {
-     const type = contentType || 'application/octet-stream';
-     const a = document.createElement('a');
-     const blob = new Blob([ content ], { type: type });
-
-     a.href = window.URL.createObjectURL(blob);
-     a.download = filename;
-     a.click();
-   }
-
-   clearLocalStorage = () => {
-     localStorage.clear();
-     console.info('cleared local storage');
-   }
-
-   handleAnalyticsEvents = (event) => {
-     this.setState({ analyticsEvents: [ ...this.state.analyticsEvents, event ] });
-   }
-
-   handleChangeTranscriptTitle = (newTitle) => {
-     this.setState({
-       title: newTitle
-     });
-   }
-
-   handleChangeTranscriptName = (value) => {
-     this.setState({ fileName: value });
-   }
-
-   render() {
-
-     return (
-       <div className={ style.container }>
-         <span>
-            Demo page for <mark>React Transcript Editor</mark> - Component |{' '}
-           <a
-             href="https://github.com/bbc/react-transcript-editor"
-             rel="noopener noreferrer"
-             target="_blank"
-           >
+  download = (content, filename, contentType) => {
+    const type = contentType || 'application/octet-stream';
+    const a = document.createElement('a');
+    const blob = new Blob([ content ], { type: type });
+
+    a.href = window.URL.createObjectURL(blob);
+    a.download = filename;
+    a.click();
+  };
+
+  clearLocalStorage = () => {
+    localStorage.clear();
+    console.info('cleared local storage');
+  };
+
+  handleAnalyticsEvents = event => {
+    this.setState({ analyticsEvents: [ ...this.state.analyticsEvents, event ] });
+  };
+
+  handleChangeTranscriptTitle = newTitle => {
+    this.setState({
+      title: newTitle
+    });
+  };
+
+  handleChangeTranscriptName = value => {
+    this.setState({ fileName: value });
+  };
+
+  render() {
+    return (
+      <div className={ style.container }>
+        <span>
```

Comment:
This span doesn't look great. Can make the whole thing a link instead of GitHub Repo as a separate url -

or alternatively a github icon linking to it. Also maybe center text.

I kinda think it's not necessary at all in it's current form so happy to chuck it 🤷‍♂️

Reply:
I've replaced it with the github icon, and removed the span. But we need something saying that that page is a demo otherwise some users might get confused etc..
+ good to link to the repo in case someone finds the project through the demo page etc..